test/test.hs

```diff
@@ -411,13 +411,10 @@ tests = testGroup "hevm"
     , testProperty "simpProp-equivalence-sym" $ \(LitProp p) -> ioProperty $ do
         let simplified = pand (Expr.simplifyProps [p])
         checkEquivProp simplified p
-    -- This would need to be a fuzz test I think. The SMT encoding of Keccak is not precise
-    -- enough for this to succeed
-    , ignoreTest $ testProperty "storage-slot-simp-property" $ \(StorageExp s) -> ioProperty $ do
-        T.writeFile "unsimplified.expr" $ formatExpr s
-        let simplified = Expr.simplify s
-        T.writeFile "simplified.expr" $ formatExpr simplified
-        checkEquiv simplified s
+    , testProperty "storage-slot-simp-property" $ \(StorageExp s) -> ioProperty $ do
+        let s2 = Expr.structureArraySlots s
```

Comment:
Note: this is needed, otherwise, the tests fail due to inability for the system to reverse some Keccak values. While it could be fixed without this hack, it would involve writing a minimal simplifier, I believe. That would likely be more error-prone than trusting `structureArraySlots`, which I think is something we kinda have to trust anyway, and is quite straightforward.

Reply:
I'm fine with this, but maybe just add a comment explaining what's going on?

Reply:
Good point, thanks, added an explanation :)